models/user.go

```diff
@@ -945,13 +945,23 @@ func deleteUser(e *xorm.Session, u *User) error {
 	// ***** END: Star *****
 
 	// ***** START: Follow *****
+	followees := make([]*Follow, 0, 10)
```

Comment:
@lafriks I have an updated version, but it's blocked on a fix to xorm (https://github.com/go-xorm/builder/pull/4)